ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java

```diff
@@ -86,6 +87,21 @@ public void connect(final String url, @Nullable final ReadableArray protocols, @
       }
     }
 
+    if (protocols != null && protocols.size() > 0) {
+      StringBuilder protocolsValue = new StringBuilder("");
+      for (int i = 0; i < protocols.size(); i++) {
+        String v = protocols.getString(i).trim();
+        if (!TextUtils.isEmpty(v) && !v.contains(",")) {
+          protocolsValue.append(v);
+          protocolsValue.append(",");
+        }
+      }
+      if (!TextUtils.isEmpty(protocolsValue)) {
+        protocolsValue.replace(protocolsValue.length() - 1, protocolsValue.length(), "");
```

Comment:
what do you think?

**currently, I prefer to:**

1. keep using `StringBuilder` like another module. 
2. remove `import TextUtils`, and use `String.isEmpty()` instead of `TextUtils.isEmpty()` --- I don't think it's necessary to support devices before Android 2.3 

Or, if the import of TextUtils won't hurt anything, just keep using it and there is no need to update this PR.

Correct me if I'm wrong. thanks!!